server/src/test/java/org/candlepin/service/impl/stub/StubEntitlementCertServiceAdapter.java

```diff
@@ -87,4 +91,31 @@ public EntitlementCertificate generateUeberCert(Entitlement entitlement,
         return generateEntitlementCert(entitlement, product);
     }
 
+    @Override
+    public Map<String, EntitlementCertificate> generateEntitlementCerts(Consumer consumer,
+            Map<String, Entitlement> entitlements, Map<String, Product> products)
+        throws GeneralSecurityException, IOException {
+
+        Map<String, EntitlementCertificate> result = new HashMap<String, EntitlementCertificate>();
+        for (Entry<String, Entitlement> entry : entitlements.entrySet()) {
+            EntitlementCertificate cert = generateEntitlementCert(entry.getValue(),
+                    products.get(entry.getKey()));
+            result.put(entry.getKey(), cert);
+        }
+        return result;
+    }
+
+    @Override
+    public Map<String, EntitlementCertificate> generateUeberCerts(Consumer consumer,
+            Map<String, Entitlement> entitlements, Map<String, Product> products)
+        throws GeneralSecurityException, IOException {
```

Comment:
Rather than specifying the exceptions, maybe just `throws Exception`.  That's what we do elsewhere and it cuts down on boilerplate and additional imports.  Since the exception isn't recoverable, there's not much point in keeping it strongly typed.